schema/json.go

```diff
@@ -63,6 +63,19 @@ func (dst *JSON) Set(src interface{}) error {
 		} else {
 			*dst = JSON{Bytes: value, Status: Present}
 		}
+	// In the case of map[string]string (fields like 'Tags'), an empty map is easier for users to work with than
+	// postgres "null", or JSON "null".
+	case map[string]string:
```

Comment:
I think this is not generic enough because it can be `map[string]interface` and can be []interface{} and can be `[]string` all valid jsons.

we should then solve it on the `default` i.e to check if null and then make it `{}`

Reply:
PTAL at new version - more generic